frontend/src/components/my-links/RightPanel.tsx

```
@@ -0,0 +1,980 @@
+  return (
+    <div 
+      ref={itemRef}
+      className={`group relative flex items-center p-3 rounded-xl transition-all cursor-pointer border ${(isBulkEditMode && isSelected) || (isTitleEditing || isNoteEditing || isTagEditing) ? 'bg-purple-50/50 dark:bg-purple-900/10 border-purple-200 dark:border-purple-800' : 'border-transparent hover:border-purple-200/50 dark:hover:border-purple-800/50 hover:bg-purple-50/60 dark:hover:bg-purple-900/10'}`}
+      onClick={() => {
+        if (isBulkEditMode && onToggleSelect) {
+          onToggleSelect(data.bookmarkId);
+        } else if (!isNoteEditing && !isTitleEditing && data.link?.originalUrl) {
+          window.open(data.link.originalUrl, '_blank', 'noopener,noreferrer');
+        }
+      }}
+    >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)